pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -1309,7 +1291,15 @@ func doNetworkInstanceDelete(
 		}
 		DNSStopMonitor(status.BridgeNum)
 	}
+	if status.BridgeMac != "" {
```

Comment:
Shouldn't you also check that BridgeIPAddr was set? Otherwise there will be a spurious error logged by releaseIPv4FromNetworkInstance.